packages/tupaia-web/src/features/Dashboard/DashboardMenu/SubscribeModal.tsx

```diff
@@ -3,185 +3,141 @@
-import { Modal, Form, TextField, Title, ModalParagraph } from '../../../components';
-import { FORM_FIELD_VALIDATION, MOBILE_BREAKPOINT } from '../../../constants';
+import { DialogActions, Typography } from '@material-ui/core';
+import { SpinningLoader, Button as UIButton } from '@tupaia/ui-components';
+import { useUser } from '../../../api/queries';
+import { Modal, Form, TextField, ModalParagraph } from '../../../components';
+import { FORM_FIELD_VALIDATION } from '../../../constants';
 import { useSubscribeDashboard, useUnsubscribeDashboard } from '../../../api/mutations';
 import { useDashboardMailingList, useDashboard } from '../utils';
-import {
-  MODAL_SUBSCRIBE_TEXT,
-  MODAL_SUBSCRIBE_TITLE,
-  MODAL_UNSUBSCRIBE_TEXT,
-  MODAL_UNSUBSCRIBE_TITLE,
-} from './constants';
 
-const Wrapper = styled.div`
-  width: 45rem;
-  max-width: 100%;
-  padding: 2.5rem 1.875rem 0rem 1.875rem;
-  display: flex;
-  justify-content: center;
-`;
-
-const Container = styled.div`
+const ModalForm = styled(Form)`
   display: flex;
   flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  flex-basis: 83.3333%;
-  button {
-    text-transform: none;
-    padding: 0.5em 1.75em;
-    font-size: 1rem;
-  }
-  .loading-screen {
-    background-color: ${props => props.theme.palette.background.default};
-    border: 0;
-    button {
-      padding: 0.5em 1.75em;
-      font-size: 1rem;
-    }
-  }
-`;
-
-export const ErrorMessageText = styled(Typography)`
-  font-size: 0.9rem;
-  line-height: 1.3;
-  margin-top: 1rem;
-  margin-bottom: 2rem;
-`;
-
-const ButtonGroup = styled.div`
-  padding-top: 2.5rem;
-  width: 100%;
-  display: flex;
-  justify-content: flex-end;
+  justify-content: space-between;
+  width: 42rem;
+  max-width: 100%;
+  min-height: 18rem;
+  padding: 3rem 1.5rem 0.5rem;
+  text-align: left;
 `;
 
-const SubscribeButton = styled(Button)`
-  margin-left: 1.2rem;
+const Button = styled(UIButton)`
+  text-transform: none;
+  font-size: 0.875rem;
+  padding: 0.3rem 1.1rem;
 `;
 
 const EmailInput = styled(TextField)<{ $disabled?: boolean }>`
-  width: 50%;
-  margin: auto;
-  @media screen and (max-width: ${MOBILE_BREAKPOINT}) {
-    width: 100%;
-  }
-
+  width: 21rem;
+  max-width: 100%;
+  margin: 0 auto 2rem;
   ${({ $disabled, theme }) =>
-    $disabled
-      ? `
-
-      .MuiInput-input {
-        color: ${theme.palette.text.secondary}; 
-      }
-        
-      .MuiInput-underline:before { border-bottom: 1px outset rgba(255, 255, 255, 0.7); transition: none;  }
-      .MuiInput-underline:hover:before { border-bottom: 1px outset rgba(255, 255, 255, 0.7); }
-      .MuiInput-underline:after { border-bottom-style: none; }  
-      `
-      : ''};
+    $disabled &&
+    `.MuiInput-input {
+    color: ${theme.palette.text.secondary};
+  }
+  .MuiInput-root:before,.MuiInput-root:hover:before, .MuiInput-root:after {
+    border-bottom: 1px solid ${theme.palette.divider};
+  }`}
```

Comment:
Much nicer! Would be nice to use a `css` tagged template literal so we get formatting for free too

Reply:
I didn't know you could do that but love it!